frontend/front/src/pages/Public/Components/PublicSurvey.js

```diff
@@ -4,10 +4,12 @@ import SurveyComponent from "../../../components/SurveyComponent/SurveyComponent
 const PUBLIC_SURVEY_ID = "1";
 
 export const PublicSurvey = forwardRef((props, ref) => (
-  <SurveyComponent
-    {...props}
-    surveyId={PUBLIC_SURVEY_ID}
-    formSpacing={5}
-    ref={ref}
-  />
+  <div ref={ref}>
```

Comment:
If you fixed the ref thing at the SurveyComponent level I don't think you need this div any more. Not sure what the behavior is if the ref is set to two different divs.